float64 wind_x
float64 wind_y
float64 wind_z
